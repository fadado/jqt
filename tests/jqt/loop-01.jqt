<meta name='author' content='{{.author[]}}' />
