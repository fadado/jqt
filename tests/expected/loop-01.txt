<meta name='author' content='Anonymous' />
<meta name='author' content='Unknow' />
